### ISSUE TITLE
Add Review Us Nag Dialog

### ISSUE THREAD

**Issue by bkerensa** [Glucosio/glucosio-android#237]
We should add a review us nag dialog:
![](https://files.slack.com/files-tmb/T09S6EY0J-F1EKA2FDY-ef12f5c1a2/screenshot_20160606-160609_1024.png)

Requirements:
- User has opened app five times
- User has made at least five entries of any metric type
- Once dialog is displayed it is not displayed again 

This will be on hold by a separate creative bug opening with @elioqoshi 
once we have creative will updated this bug. If you begin work on this bug
please assign yourself.

**Comment by PaoloRotolo** [comment]
I'm assigning this to design team for now. Ping me back when a final design is ready to be coded :)

**Comment by PaoloRotolo** [comment]
@elioqoshi @satya164 any updates on this?

**Comment by satya164** [comment]
Will get to this tonight.

**Comment by bkerensa** [comment]
Lets make sure we do not use any prohibited language in asking for reviews as I know there are some rules around what and how you say it. 

I will look into the rules on language today

**Comment by satya164** [comment]
Here is an idea,

![11-rate-us-dialog](https://cloud.githubusercontent.com/assets/1174278/17124907/428cd10e-530c-11e6-8b09-a1b28ea281d9.png)

We ask for rating, if user gives 4-5 starts, we redirect to play store, if he gives a low rating, we ask for feedback

![12-feedback-dialog](https://cloud.githubusercontent.com/assets/1174278/17124924/5c83af9c-530c-11e6-8979-d083d70dddba.png)

This avoids lower ratings on playstore as well as helps us improve Glucosio better.

**Comment by satya164** [comment]
Not exactly review us dialog, but interestin - https://medium.com/budi-brain/designing-in-app-survey-6163304e88dd#.q401wpikn

**Comment by bkerensa** [comment]
@elioqoshi feedback?

**Comment by elioqoshi** [comment]
Checking out and working on a design

**Comment by bkerensa** [comment]
Just one more thought on this we should seperate feedback from rate in app store as I do not want the user to be confused we are trying to deter bad feedback to a feedback form which could violate Google Play rules anyways. 

And I think Elio can make us a slick design :)

**Comment by elioqoshi** [comment]
![review 1](https://cloud.githubusercontent.com/assets/5436686/17204912/7078df52-54a8-11e6-8c2a-305fe3e3f774.jpg)
![review 2](https://cloud.githubusercontent.com/assets/5436686/17204913/7079f734-54a8-11e6-8400-32e7f1267c2c.jpg)
![review 3](https://cloud.githubusercontent.com/assets/5436686/17204911/70783dd6-54a8-11e6-8c90-afbbf915be0e.jpg)

I think the 3rd one is the best option here. What do you think?

**Comment by bkerensa** [comment]
I like the 3rd one too and of course @PaoloRotolo when we implement this we need some logic so it is not nagging them more than once. We should nag only once maybe on 5th time they make an entry after two opens.

**Comment by emartynov** [comment]
@satya164: what I see most of apps using trick like “Do you enjoy app?” and when user press “yes” they show dialog about rating

**Comment by satya164** [comment]
@emartynov Yeah, that could work too.

**Comment by bkerensa** [comment]
@PaoloRotolo will this be ready for Monday code freeze or is it too late?

**Comment by PaoloRotolo** [comment]
I can try, I'm also working on the new bottom sheet menu
